app/models/person.rb

```diff
@@ -92,4 +94,33 @@ def picture_size
 
     errors.add(:picture, 'grösse kann maximal 10MB sein')
   end
+
+  class << self
+    def from_omniauth(auth) # rubocop:disable Metrics/AbcSize
+      person = where(email: auth.info.email).first_or_create do |user|
+        user.name = auth.info.name
+        user.birthdate = DateTime.new(2020, 1, 1)
+        user.nationality = 'CH'
+        user.location = 'Schweiz'
+        user.title = 'Software Engineer'
+        user.marital_status = :single
+        user.company = Company.first
+      end
+      set_admin(person, auth)
+    end
+
+    private
+
+    def set_admin(person, auth)
+      person.is_admin = admin?(auth)
+      person.save
```

Comment:
What if we remove the admin flag from a person in the keycloak interface?
What if the person logs in and that person already has a profile? (will be the case on PROD)